cypress/support/step-definitions/common-steps.js

```diff
@@ -397,3 +397,43 @@ Then('{string} tab in {string} tab list is visible', (knobsName, position) => {
     default: throw new Error('There are only two tab list elements on the page');
   }
 });
+
+When('I press ESC in iFrame', () => {
+  cy.focused().trigger('keydown', { keyCode: 16, which: 16, release: false });
+  cy.focused().trigger('keydown', { keyCode: 27, which: 27 });
+});
+
+When('I press Tab in iFrame', () => {
```

Comment:
These steps have no reference to iframe, please update them `I press Tab` could be used in many places without iframe.

Reply:
Same with the other steps in this file

Reply:
agree, fixed